de.klayout.KLayout.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>de.klayout.KLayout</id>
+  
+  <name>KLayout</name>
+  <summary>KLayout, viewer and editor for mask layouts</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  
+  <description>
+    <p>
+      Mask layout viewer and editor for the chip design engineer.
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">org.zotero.KLayout.desktop</launchable>
```

Comment:
That doesn't look right?

Reply:
Oops, I copied that over from the appdata file which I modeled on the Zotero flatpak. I will fix it after we resolve the app ID question.